docs/content/latest/faq/general.md

```diff
@@ -378,6 +378,10 @@ See [YugabyteDB in Comparison](../../comparisons/)
 - [Apache Cassandra](../../comparisons/cassandra/)
 - [Azure Cosmos DB](../../comparisons/azure-cosmos/)
 
+## What is the difference between `ysqlsh` and `psql`?
+
+The YSQL shell (`ysqlsh`) is functionally similar to PostgreSQL's `psql` , but uses different default values for some variables (for example, the default user, default databasse, and the path to SSL certificates). This is done for the user's convenience. You can work on the same host with PostgreSQL by using `psql` and with your YugabyteDB cluster by using `ysqlsh`. For more details, see [ysqlsh](../../admin/ysqlsh). In the Yugabyte `bin` directory, there is a deprecated `psql` alias that opens the `ysqlsh` CLI.
```

Comment:
The line "You can work on the same host with PostgreSQL by using `psql` and with your YugabyteDB cluster by using `ysqlsh`." is confusing - I think we can remove it and simply state that bin/ has a deprecated psql binary which is an alias to ysqlsh.